gears-of-fortune.html

```diff
@@ -0,0 +1,29 @@
+@@include('./partials/header.html')
+
+<div class="section section-with-top">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-12">
+        <h1>Gears of fortune</h1>
+
+        <!-- Start Gamevy Embed code -->
+        <div class="game-embed game-embed--the-heist"
+             data-gamevy-embed
+             data-operator="lottoland"
```

Comment:
That might have been my fault, the other ones probably need updating as well. What should we use for the homepage? gamevy?